sdk/table/azure-table/README.md

```diff
@@ -0,0 +1,354 @@
+# Azure Table client library for Python
+
+Azure Table is a service for storing data that can be accessed from anywhere in the world via authenticated calls using HTTP or HTTPS. 
+
+Common uses of Azure Table include:
+
+* Storing structured data in the form of tables
+* Querying tables and entities 
+
+[Source code](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-queue/azure/storage/queue) | [Package (PyPI)](https://pypi.org/project/azure-storage-queue/) | [API reference documentation](https://aka.ms/azsdk-python-storage-queue-ref) | [Product documentation](https://docs.microsoft.com/azure/storage/) | [Samples](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-queue/samples)
+
```

Comment:
The above links still point to queues. We should be able to predict what the URLs will look like for tables. For the API reference documentation, since we don't have a link to published docs yet you can use this link I just created:

https://aka.ms/azsdk/python/table/docs

When docs are published, we will redirect that link to your docs.